modules/gapi/perf/common/gapi_render_perf_tests_inl.hpp

```diff
@@ -0,0 +1,827 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#include "gapi_render_perf_tests.hpp"
+
+namespace opencv_test
+{
+
+namespace {
+void create_rand_mats(const cv::Size &size, MatType type, cv::Mat &ref_mat, cv::Mat &gapi_mat)
+{
+    ref_mat.create(size, type);
+    cv::randu(ref_mat, cv::Scalar::all(0), cv::Scalar::all(255));
+    ref_mat.copyTo(gapi_mat);
+};
+
+} // namespace
+
+PERF_TEST_P_(RenderTestFTexts, RenderFTextsPerformanceBGROCVTest)
+{
+    std::wstring text;
+    cv::Size sz;
+    cv::Point org;
+    int fh = 0;
+    cv::Scalar color;
+    cv::GCompileArgs comp_args;
+    std::tie(text ,sz ,org ,fh ,color, comp_args) = GetParam();
+
+    MatType type =  CV_8UC3;
+    cv::Mat gapi_mat, ref_mat;
+    create_rand_mats(sz, type, ref_mat, gapi_mat);
+
+    // G-API code //////////////////////////////////////////////////////////////
+    cv::gapi::wip::draw::Prims prims;
+    prims.emplace_back(cv::gapi::wip::draw::FText{text, org, fh, color});
+
+    cv::GMat in;
+    cv::GArray<cv::gapi::wip::draw::Prim> arr;
+
+    cv::GComputation comp(cv::GIn(in, arr),
+                          cv::GOut(cv::gapi::wip::draw::render3ch(in, arr)));
+
+    // Warm-up graph engine:
+    comp.apply(gin(gapi_mat, prims), gout(gapi_mat), std::move(comp_args));
+
+    TEST_CYCLE()
+    {
+        comp.apply(gin(gapi_mat, prims), gout(gapi_mat), std::move(comp_args));
```

Comment:
`comp_args` has been already moved.   
Check test for `CPU`:  https://github.com/opencv/opencv/blob/master/modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp#L42